client/src/components/topBanner/topBanner.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
If you call this file index.js instead of topBanner.js, then you can import it with

`import TopBanner from 'components/topBanner';`